calibrate-touchscreen/src/main.rs

```diff
@@ -0,0 +1,429 @@
+use std::{fs::File, os::unix::prelude::AsFd};
+use std::cmp::max;
+use std::collections::{HashMap};
+
+use wayland_client::{delegate_noop, protocol::{
+    wl_buffer, wl_compositor, wl_keyboard, wl_registry, wl_seat, wl_shm, wl_shm_pool,
+    wl_surface, wl_output, wl_touch, wl_pointer
+}, Connection, Dispatch, QueueHandle, WEnum, Proxy};
+
+use wayland_protocols::xdg::shell::client::{xdg_surface, xdg_toplevel, xdg_wm_base};
+
+fn main() {
```

Comment:
```suggestion
fn main() -> Result<(), Box<dyn Error>>{
```

This'll let you use the error-forwarding `?` operator instead of `unwrap()`.

Reply:
(Will also require a `use std::error::Error` above)